app/src/main/java/com/hover/stax/database/Migrations.kt

```diff
@@ -220,5 +220,16 @@ class Migrations {
             database.execSQL("CREATE UNIQUE INDEX IF NOT EXISTS `index_accounts_name_sim_subscription_id` ON `accounts` (`name`, `sim_subscription_id`)")
```

Comment:
I think the above create/insert calls above may need to be re-done but with the default statement added for `institutionId`?

Reply:
That's what I see - no other way to add a default value to it